src/systemctl/systemctl.c

```diff
@@ -5467,6 +5476,8 @@ static int show_one(
                 { "TasksMax",                       "t",              NULL,           offsetof(UnitStatusInfo, tasks_max)                         },
                 { "IPIngressBytes",                 "t",              NULL,           offsetof(UnitStatusInfo, ip_ingress_bytes)                  },
                 { "IPEgressBytes",                  "t",              NULL,           offsetof(UnitStatusInfo, ip_egress_bytes)                   },
+                { "IOReadBytes",                    "t",              NULL,           offsetof(UnitStatusInfo, io_read_bytes)                     },
+                { "IOWriteBytes",                   "t",              NULL,           offsetof(UnitStatusInfo, io_write_bytes)                    },
```

Comment:
Here too, "IORead/IOWrite" → "Input/Output" or "DiskRead/DiskWrite"?

Reply:
As-is appears more consistent with e.g. "IPIngresBytes" "IPEgressBytes", "IP"/"IO" being the parent category of the stats. /bikeshed

Reply:
Regarding the suggestion to use "Disk" as prefix here: 

So far "IO" was used as common prefix for settings/props related to the "io" cgroupv2 controller, and I think that makes sense to stick to, given that this is the same data. Yes, "io" is unfortunately named, but it is how it is. Or to say this differently "IOAccounting=" would have to be renamed to "DiskAccounting=" too, if we'd rename IOReadBytes to DiskReadBytes= I think, and I'd avoid that...

Moreover I'd like to add disk space accounting to services too (i.e. minimal quota hookup so that you can see the space taken up by StateDirectory= and friends in `systemctl status` output too). And I think it would make sense to call such accounting "DiskAccounting=" ultimately, i.e. I'd like to leave the "Disk" prefix for quota stuff later on.

regarding the issue of input/output vs. read/write vs. ingress/egress. I think the latter is primarily used by networking people (and only newer networking code that is), and I think we should probably use it for that only. And as mentioned earlier I think the read/write naming is a bit clearer than input/output since it's not as ambiguously dependent from the PoV of the beholder.

Most importantly though, we already have IOReadBandwidthMax=, IOWriteBandwidthMax= settings currently, and thus I think we should stick to the IORead…/IOWrite… prefix for these concepts to make things least confusing.

Reply:
Ack, let's keep "IO" prefix.